README.md

```diff
@@ -13,6 +13,15 @@
 **HTTPlug, the HTTP client abstraction for PHP.**
 
 
+## Intro
+
+HTTPlug served as a reference is the predecessor of [PSR-18](https://github.com/php-fig/fig-standards/blob/master/accepted/http-client/http-client.md)
```

Comment:
and we should mention that version 2 of httplug extends the psr-18 interfaces to allow a convenient migration path.